src/main/java/net/dries007/tfc/mixin/client/BreakingItemParticleMixin.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed under the EUPL, Version 1.2.
+ * You may obtain a copy of the Licence at:
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ */
+
+package net.dries007.tfc.mixin.client;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.multiplayer.ClientLevel;
+import net.minecraft.client.particle.BreakingItemParticle;
+import net.minecraft.client.particle.TextureSheetParticle;
+import net.minecraft.world.item.ItemStack;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(BreakingItemParticle.class)
+public abstract class BreakingItemParticleMixin extends TextureSheetParticle {
+    protected BreakingItemParticleMixin(ClientLevel p_108323_, double p_108324_, double p_108325_, double p_108326_) {
```

Comment:
please name parameters and put braces on newlines

Reply:
oop, just fixed